.github/workflows/trivy-release.yml

```
@@ -0,0 +1,199 @@
+        uses: step-security/harden-runner@8d3c67de8e2fe68ef647c8db1e6a09f647780f40 # v2.19.0
+        with:
+          egress-policy: audit
+
+      - name: Install Trivy
+        uses: aquasecurity/setup-trivy@3fb12ec12f41e471780db15c232d5dd185dcb514 # v0.2.6
+        with:
+          version: v0.68.2
+          cache: true
+
+      - name: Scan Image (JSON)
+        run: |
```

actionlint (1.7.12)

[error] property "image_digest" is not defined in object type {criticality: string; exposure: string; image-digest: string; image_repo: string; name: string}

(expression)